resource/supported.go

```diff
@@ -111,6 +114,7 @@ var (
 		S3Bucket:            "Name",
 		SecurityGroup:       "GroupId",
 		Subnet:              "SubnetId",
+		Trail:               "TrailName",
```

Comment:
```suggestion
		Trail:               "Name",
```

Reply:
Adjusted; I am slowly understanding what needs to get modified and why.